pkg/packagekit/kpatch.jsx

```diff
@@ -0,0 +1,376 @@
+                               </>}
+        />;
+
+        return (<>
+            <div id="kpatch-settings">
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <Flex grow={{ default: 'grow' }} alignItems={{ default: 'alignItemsBaseline' }}>
+                        <FlexItem>
+                            <b>{_("Kernel patching")}</b>
+                        </FlexItem>
+                        <FlexItem>
+                            {state}
+                        </FlexItem>
+                    </Flex>
+                    <Flex>
+                        <Button variant="secondary"
+                                isSmall
+                                disabled={!this.props.privileged}
+                                onClick={action}>
+                            {actionText}
+                        </Button>
+                    </Flex>
+                </Flex>
+            </div>
+            <Modal position="top" variant="small" id="kpatch-setup" isOpen={this.state.showModal}
+                title={_("Kernel patch settings")}
+                onClose={ this.onClose }
+                footer={
+                    <>
+                        <Button variant="primary"
+                                isLoading={ this.state.updating }
+                                isDisabled={ this.state.updating }
+                                onClick={ this.handleChange }>
+                            {_("Apply")}
+                        </Button>
+                        <Button variant="link"
+                                isDisabled={ this.state.updating }
+                                onClick={ this.onClose }>
+                            {_("Cancel")}
+                        </Button>
+                    </>
+                }>
+                <>
+                    {error}
+                    {body}
+                </>
+            </Modal>
+        </>);
+    }
+}
+
+KpatchSettings.propTypes = {
+    privileged: PropTypes.bool.isRequired,
+};
+
+export class KpatchStatus extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            loaded: [],
+            installed: [],
+            changelog: null, // FIXME - load changelog
+            opened: false,
+        };
+
+        this.checkSetup = this.checkSetup.bind(this);
+    }
+
+    checkSetup() {
+        cockpit.spawn(["kpatch", "list"], { superuser: "try", err: "ignore" })
+                .then(m => {
+                    const parts = m.trim().split("\n\n");
+                    if (parts.length !== 2 ||
+                        !parts[0].startsWith("Loaded patch modules:") ||
+                        !parts[1].startsWith("Installed patch modules:")) {
```

Comment:
Ugh.. Let's please at least call kpatch with `LC_MESSAGES=C.UTF-8` then, to not stumble over translations. Do our integration tests catch this? (As a lot of them use the mocked `kpatch list`)